README.md

```diff
@@ -4,28 +4,40 @@ ChitChat is a Python-powered Social Network.
 
 ## Installation
 
-**TODO** Update the sections below:
+1. cd into backend folder
+    cd backend
```

Comment:
Let's add markdown formatting for this:

i.e \`\`\`cd backend\`\`\`

Reply:
Noted.